app/services/taxonomy/fetch_all_taxons.rb

```diff
@@ -1,5 +1,5 @@
 module Taxonomy
-  class TaxonFetcher
+  class FetchAllTaxons
```

Comment:
I'm not too sure about this one, as this class does more than just fetch all taxons. Not super convinced of the following suggestion, but here it goes:

```
module Taxonomy
  class RemoteTaxons
    def all
       # existing taxons method
    end

    def content_ids
      # existing taxon_content_ids
    end

    def parents_for(taxon)
      # existing parents_for_taxon
    end
  end
end
```

WDYT?